packages/contracts-bedrock/src/dispute/weth/DelayedWETH.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+
+import { IDelayedWETH } from "src/dispute/interfaces/IDelayedWETH.sol";
+import { IWETH } from "src/dispute/interfaces/IWETH.sol";
+import { WETH98 } from "src/dispute/weth/WETH98.sol";
+
+import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
+
+/// @title DelayedWETH
+/// @notice DelayedWETH is an extension to WETH9 that allows for delayed withdrawals. Accounts must
+///         trigger an unlock function before they can withdraw WETH. Accounts must trigger unlock
+///         by specifying a sub-account and an amount of WETH to unlock. Accounts can trigger the
+///         unlock function at any time, but must wait a delay period before they can withdraw
+///         after the unlock function is triggered. DelayedWETH is designed to be used by the
+///         DisputeGame contracts where unlock will only be triggered after a dispute is resolved.
+///         DelayedWETH is meant to sit behind a proxy contract and has an owner address that can
+///         pull WETH from any account and can recover ETH from the contract itself. Variable and
+///         function naming vaguely follows the vibe of WETH9. Not the prettiest contract in the
+///         world, but it gets the job done.
+contract DelayedWETH is OwnableUpgradeable, WETH98, IDelayedWETH, ISemver {
+    /// @notice Semantic version.
+    /// @custom:semver 0.2.0
+    string public constant version = "0.2.0";
+
+    /// @inheritdoc IDelayedWETH
+    mapping(address => mapping (address => WithdrawalRequest)) public withdrawals;
+
+    /// @notice Withdrawal delay in seconds.
+    uint256 internal immutable DELAY_SECONDS;
```

Comment:
Is DelayedWeth a singleton for a given superchain, or is there one per OP Chain? 
I'm assuming the owner/sub-account arch means it is a singleton.

Reply:
One per OP Chain at the moment, because of the ownershop/proxy structure.